src/main/api/Abilities.cpp

```diff
@@ -0,0 +1,141 @@
+/*
+ * MacroQuest: The extension platform for EverQuest
+ * Copyright (C) 2002-2023 MacroQuest Authors
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License, version 2, as published by
+ * the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+
+#include "pch.h"
+
+#include "mq/api/Abilities.h"
+
+#include "eqlib/PcClient.h"
+#include "eqlib/EQClasses.h"
+
+// Defined in MQ2Globals.cpp but we don't want to bring in all of MQ2Globals.h
+MQLIB_VAR const char* szInnates[];
+
+namespace mq {
+
+int GetAdjustedSkill(int nSkill)
+{
+	return pLocalPC ? pLocalPC->GetAdjustedSkill(nSkill) : 0;
+}
+
+int GetBaseSkill(int nSkill)
+{
+	return pLocalPC ? pLocalPC->GetBaseSkill(nSkill) : 0;
+}
+
+bool HasSkill(int nSkill)
+{
+	return pLocalPC && pLocalPC->HasSkill(nSkill);
+}
+
+bool SkillIsActivatable(int nSkill)
+{
+	return nSkill < NUM_SKILLS && pSkillMgr && pSkillMgr->pSkill[nSkill]->Activated;
+}
```

Comment:
these could be inline?

Reply:
Moved these into the header and inlined them.